packages/react-chat/package.json

```diff
@@ -42,12 +42,12 @@
     "test:unit": "yarn g:vitest run --coverage"
   },
   "dependencies": {
-    "@voiceflow/base-types": "2.97.3",
+    "@voiceflow/base-types": "2.111.4",
```

Comment:
latest versions of `base-types` and `voiceflow-types` depend on a version of `@voiceflow/common` that correctly includes the `.js` file extensions in import statements